src/Presentation/Presenters/ExceptionHtmlPresenterInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Presentation\Presenters;
+
+use WMDE\Fundraising\Frontend\Presentation\TwigTemplate;
+
+interface ExceptionHtmlPresenterInterface {
+	public function setTemplate( TwigTemplate $template ): void;
```

Comment:
Why do you need the template injection? It carries the risk of creating invalid presenters if you forget to call it (and I think you did). `getLayoutTemplate` of FundraisingForntend is public and can be called from the environment factories.

IMO, constructor injection is always preferrable to setter injection.